sdk/storage/storage-queue/README.md

```diff
@@ -1,30 +1,42 @@
-# Azure Storage client library for JavaScript - Queue
+# Azure Queue Storage client library for JavaScript
 
 Azure Queue storage provides cloud messaging between application components. In designing applications for scale, application components are often decoupled, so that they can scale independently. Queue storage delivers asynchronous messaging for communication between application components, whether they are running in the cloud, on the desktop, on an on-premises server, or on a mobile device. Queue storage also supports managing asynchronous tasks and building process work flows.
 
-This project provides a client library in JavaScript that makes it easy to consume Microsoft Azure Queue Storage service.
+This project provides a client library in JavaScript that makes it easy to consume the Azure Queue Storage service.
 
-Version: 12.0.0-preview.5
+Use the client libraries in this package to:
+- Get/Set Queue Service Properties
+- Create/List/Delete Queues
+- Enqueue/Dequeue/Peek/Clear/Update/Delete Queue Messages
 
-- [Package (npm)](https://www.npmjs.com/package/@azure/storage-queue/v/12.0.0-preview.5)
-- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue/samples)
-- [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/storage-queue/index.html)
-- [Product documentation](https://docs.microsoft.com/azure/storage/queues/storage-queues-introduction)
-- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue)
-- [Azure Storage Queue REST APIs](https://docs.microsoft.com/rest/api/storageservices/queue-service-rest-api)
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue) |
+[Package (npm)](https://www.npmjs.com/package/@azure/storage-queue/v/12.0.0-preview.5) |
```

Comment:
We don't need to have the version number in the link since we are going GA.

```suggestion
[Package (npm)](https://www.npmjs.com/package/@azure/storage-queue) |
```